patientIdentifierType.md

```diff
@@ -0,0 +1,112 @@
+* required: a true/false whether every patient MUST have this type
+
+* checkDigit: a true/false whether this identifier has a checkdigit at the end
+
+* validator: full class name of an [IdentifierValidator](https://docs.openmrs.org/doc/org/openmrs/patient/IdentifierValidator.html) (e.g., [`org.openmrs.patient.impl.LuhnIdentifierValidator`](https://docs.openmrs.org/doc/org/openmrs/patient/impl/LuhnIdentifierValidator.html))
+
+* locationBehavior: "REQUIRED" if a location must be associated with the identifier; "NOT_USED" if the identifier does require a location
+
+## Available operations
+
+1. [List PatientIdentifierType resources](#List-patientidentifierType-resource)
+2. [Create a patientIdentifierType record](#create-a-patientidentifierType)
+3. [Update a patientIdentifierType record](#update-a-patientIdentifierType)
+4. [Delete a patientIdentifierType record](#delete-a-patientIdentifierType)
+
+### List PatientIdentifierType resource
+
+* #### List patientIdentifierType
+
+    Fetch all non-retired patientIdentifierTypes resources that match any specified parameters otherwise fetch all non-retired patients. Returns a `200 OK` status with the patientIdentifierType response. If the user is not logged in it returns `401 Unauthorized` status is returned.
+
+    ```console
+    GET /patientidentifiertype?
+        &q=**searchqQuery**
+    ```
+
+* #### List patientIdentifierType by UUID.
+
+    Retrieve a patientIdentifierType by its UUID. Returns a `404 Not Found` status if patientIdentifierType does not exist in the system. If the user is not logged in to perform this action, a `401 Unauthorized` status is returned.
+
+    ```console
+    GET /patientidentifiertype/:target_patientIdentifierType_uuid
+    ```
+### Create a patientIdentifierType
+
+* To create a patientIdentifierType you need to specify the below properties in the request. If you are not logged in to perform this action, a `401 Unauthorized` status is returned.
+
+    #### Properties
+
+    Parameter | Type | Description
+    --- | --- | ---
+    *name* | string | label for the identifier
+    *description* | string | a small description about the patientIdentifier
+    *format* | string | label for the identifier
+    *formatDescription* | string | a small description about the patientIdentifier
+    *required* | boolean | a true/false value
+    *checkDigit* | boolean | a true/false value
+    *validator* | string | `org.openmrs.patient.IdentifierValidator`
+    *locationBehavior* | "REQUIRED" or "NOT USED" | behavior of the location with respect to the identifier 
+    *uniquenessBehavior* | string | label for the identifier
+
+    ```console
+    POST /patientidentifiertype
+    {
+        "name": "string",
+        "description": "string",
+        "format": "string",
+        "formatDescription": "string",
+        "required": true,
+        "checkDigit": true,
+        "validator": "string",
+        "locationBehavior": "REQUIRED",
+        "uniquenessBehavior": "string"
```

Comment:
Please use real-world examples for values instead of the word "string".

Reply:
@bmamlin could you get me a real world example ?